### DISCUSSION TITLE
Electronic Flight Bag

### DISCUSSION THREAD

**Discussion by henkscheffer** [flybywiresim/aircraft#4173] [Ideas]
The EFB is great and very helpful. Help me to close doors when a ramp or gang way isn't available. It's an enrichment.
 I have a small request. Every time when I go flying and I use the SimBrief, I have to fill out my username. Is it maybe possible to pin the Username,

Thank you in advance

Henk Scheffer

**Comment by tracernz** [comment]
That will be coming. 👍

**Comment by henkscheffer** [comment]
Great.

Thanx 

Verstuurd vanaf mijn iPhone
